framework/sessions/__init__.py

```diff
@@ -171,5 +171,10 @@ def before_request():
 def after_request(response):
     if session.data.get('auth_user_id'):
         session.save()
-
+        from framework.auth.core import _get_current_user
```

Comment:
Is this necessary to avoid a circular import? If so, add a comment explaining that.